frontend/src/api/validators.ts

```diff
@@ -192,6 +193,7 @@ export const getAPIValidators = {
   move: string,
   payee_accounts: array(string),
   payee_transaction: Transaction.validator,
+  narration_transaction: Transaction.validator,
```

Comment:
This also needs to be added to the GetAPIParams interface, that's currently causing the svelte-check error

Reply:
Thanks I have updated it.